### PR TITLE
Add episode tasks

### PR DESCRIPTION
**Pull request by aboellinger**
**Problem**
* No function to query the tasks by episode 

**Solution**
* Add functions `all_tasks_for_episode` and `all_task_types_for_episode`

### CODE REVIEW

File: gazu/task.py [hunk 1]
```diff
@@ -81,6 +81,15 @@ def all_tasks_for_asset(asset):
     return sort_by_name(tasks)
 
 
+@cache
+def all_tasks_for_episode(episode):
+    """
+    Retrieve all tasks directly linked to given episode.
+    """
+    tasks = client.fetch_all("episodes/%s/tasks" % episode["id"])
```
Comment (frankrousseau) [inline gazu/task.py]: Can you use the normalize helper for the episode parameter? It allows to give either id or dict as a parameter for the episode parameter.

**Comment by aboellinger** [comment]
Fixed discussions.

For some reasons I had to close and reopen the pull request to force its update.

File: gazu/task.py (hunk 1 above)
Comment (aboellinger) [reply gazu/task.py]: Done

**Comment by frankrousseau** [comment]
Thank you for the fix and your contribution!

**Pull request merged**